Protobuf/dolittle/interaction/events.relativity/artifact.proto

```diff
@@ -9,7 +9,7 @@ package dolittle.events.relativity;
 
 import "system/guid.proto";
 
-option csharp_namespace = "Dolittle.Runtime.Events.Relativity.Protobuf";
+option csharp_namespace = "Dolittle.Runtime.Grpc.Interaction.Protobuf";
```

Comment:
Lets drop the Protobuf at the end - as that was actually inaccurate to begin with